javascript/javascript2/week1/Javascript2Project/functionality.js

```diff
@@ -86,11 +124,12 @@ function startTimer() {
   startTime = Date.now();
   timer = setInterval(() => {
```

Comment:
We have not seen that in the class, but here, a `requestAnimationFrame` would be better than a `setInterval` executed every 100ms.